example/flake.nix

```diff
@@ -88,8 +88,11 @@
           };
 
         # nix run .#ponysay up to start the process
```

Comment:
I assume `nix run .#ponysay` (without arguments) will start the TUI as well.

Reply:
BTW, you can do `nix develop -c just ex` to run the example using process-compose-flake from the current working copy.

Reply:
`nix run .#ponysay` should be the same as `nix run .#ponysay up` the same as `process-compose` is the same as `process-compose up`.

> BTW, you can do nix develop -c just ex to run the example using process-compose-flake from the current working copy.

Good to know, I'll try that. I always changed the import when I needed to try it out